proxa/Docusign-Integration
# Project interface

Added most of the projects interface (front-end and back-end)

*Test Procedures*
1. gradle bR
2. navigate to http://localhost:8080
3. you should see no projects in the list if you are not authenticated (api calls won't work for non-logged-in users)
4. authenticate with docusign as usual
5. You should see a project in the list
6. You can add/remove projects.  edit isn't implemented yet.
